stl/inc/bitset

```diff
@@ -400,7 +401,18 @@ public:
     }
 
     _NODISCARD bool all() const noexcept {
-        return count() == size();
+        if _CONSTEXPR_IF (_Bits == 0) {
+            return true;
+        }
+        for (size_t _Wpos = 0; _Wpos < _Words; ++_Wpos) {
+            if (_Array[_Wpos] != ~static_cast<_Ty>(0)) {
+                return false;
+            }
+        }

```

Comment:
We conventionally prefer to have newlines between non-chained control flow - e.g. a `for` loop followed by an `if` statement, or separate `if` statements. ("Chained control flow" is `if` followed by `else if` or `else`.)